backend/canisters/group/api/can.did

```diff
@@ -563,6 +563,34 @@ type ResetInviteCodeResponse = variant {
     ChatFrozen;
 };
 
+type FollowThreadArgs = record {
```

Comment:
Maybe just call this `FollowUnfollowThreadArgs`?

Reply:
I think this is fine and what we have done in general